torch/optim/lr_scheduler.py

```diff
@@ -190,14 +204,21 @@ def __init__(self, optimizer, step_size, gamma=0.1, last_epoch=-1):
         super(StepLR, self).__init__(optimizer, last_epoch)
 
     def get_lr(self):
+        if (self.last_epoch == 0) or (self.last_epoch % self.step_size != 0):
+            return [group['lr'] for group in self.optimizer.param_groups]
+        return [group['lr'] * self.gamma
+                for group in self.optimizer.param_groups]
+
+    def get_lr_closed_form(self):
```

Comment:
I suppose `get_lr` isn't a publicly documented function, but it would be nice to explain in English, somewhere, what exactly `get_lr_closed_form` does. You could do a Note: somewhere in this file, write your comments below a header `Note [LR closed form]` and then reference this note elsewhere.